infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/ShellScriptProvider.java

```diff
@@ -69,13 +69,13 @@ public InfrastructureProvisionResult provision(TestPlan testPlan, Script script)
         logger.info("Executing provisioning scripts...");
         try {
             Script createScript = script;
-            ShellExecutor executor = new ShellExecutor(null); //todo
+            ShellExecutor executor = new ShellExecutor(null);
```

Comment:
The todo was added because we need to set the proper working dir to shellexecutor. We currently pass null. :)